src/Symfony/Component/Mailer/Transport/Smtp/SmtpTransport.php

```diff
@@ -240,7 +240,12 @@ private function doRcptToCommand(string $address): void
         $this->executeCommand(sprintf("RCPT TO:<%s>\r\n", $address), [250, 251, 252]);
     }
 
-    private function start(): void
+    /**
+     * Manually connect to the SMTP server, will happen automatically when sending mails.
```

Comment:
I would also add a note (same for the stop method) about when this is useful. We should make it clear that this should be not called manually most of the time, except for some very specific use cases that we should describe.

Reply:
I am actually unsure about the use case for the having access to manually start the transport, so i think the right option here is to keep it as private.
Will add additional text to the stop() method tho.